packages/ui/feature-builder-form-controls/src/lib/interpolating-text-form-control/mentions-list/piece-trigger-mention-item/piece-trigger-mention-item.component.html

```diff
@@ -1,8 +1,7 @@
 <button mat-menu-item apTrackHover #hoverTracker="hoverTrackerDirective" class="!ap-px-5"
-  (click)="_stepMention.step.settings.pieceName !== CORE_SCHEDULE? expandSample? expandSample=false : expandSample=true :mentionClicked.emit(_stepMention) ">
```

Comment:
This was put for the purpose that schedule steps were not testable.

Reply:
I need to check how this looks now so I am leaving this comment for myself

Reply:
it looks fine, the test is fake and just returns an empty object, kind of a waste of time from UX time, but I get the change we can revisit later.